src/main/java/io/github/homchom/recode/Main.java

```diff
@@ -6,6 +6,8 @@
 public final class Main implements ClientModInitializer {
 	@Override
 	public void onInitializeClient() {
+		System.setProperty("java.awt.headless", "false"); // Enable AWT features
```

Comment:
This line used to be in the mod, but we removed it because Minecraft runs in headless mode for a reason. Unless I am mistaken, `/importfile` should use native code, not AWT.

Reply:
Seems like FileDialog is in fact AWT, also from my testing this works perfectly fine, don't see why its an issue

Reply:
FileDialog is AWT, but we shouldn't be using FIleDialog. It works for you because you are not on a headless machine.

Reply:
Minecraft uses their own Util class for getting the operating system (which is also their class) where they run a URI command to open a folder using a URI command, I don't see how opening a folder is related to picking a file. If it is in fact possible to use URI commands to open a file picker I would love for either you or someone else to work on fixing /importfile since I cannot figure this one out.